pkg/newlog/cmd/newlogd.go

```diff
@@ -108,8 +109,9 @@ var (
 	// from different goroutines, so in order to push the changes out of the
 	// goroutines local caches and correctly observe changed values in another
 	// goroutine sync/atomic synchronization is used. You've been warned.
-	syslogPrio = types.SyslogKernelLogLevelNum[types.SyslogKernelDefaultLogLevel]
-	kernelPrio = types.SyslogKernelLogLevelNum[types.SyslogKernelDefaultLogLevel]
+	remotelogPrio = types.SyslogKernelLogLevelNum[types.SyslogKernelDefaultLogLevel]
```

Comment:
when we will have eve-api to have new config for device remote logging settings, it will be good to handle the setting for 'not upload any device log', which is not part of the log-level strings.

Reply:
that's a good point!

tbh I was even thinking that we might need yet another log level - the local log level. because the user might not wanna store any logs on the device and only send them to the controller and that's not supported at the moment.
right now we store everything on the device because our `default.log.level` == `local.log.level`. and we also assume that the remote log level is going to be higher than `default.log.level` but we don't have any checks for it

Reply:
Agreed. We should include some settings through eve-api for defining (in project level, but can be overwritten by device configitem) local logging also. Remote logging level can not be higher than the local settings.

Reply:
and for end-to-end feature pov, the 'local' config should be per device, while the remote can be project scope. And the controller can add option timeout for the settings, e.g. turn on 'debug' for deviceA for 3 hours, controller side will remove this config after it expires.

Reply:
@europaul I think we always need to store/queue the logs locally since the device might not be connected to the controller when the log is generated. But a notion of which log levels to keep stored on the device (after sent to the controller) might be what makes sense here.

Reply:
@eriknordmark the logs that are meant to be sent to the controller are queued until they are
- sent to the controller - then they are removed
- fail to many many times - then we put them in `failedUpload`